app/src/main/res/layout/solution_summary.xml

```diff
@@ -108,6 +108,7 @@
             app:layout_constraintVertical_bias="0.0"
             app:rotationAnimationAngle="@{180f}"
             app:srcCompat="@drawable/ic_arrow_drop_down_black_24dp"
+            android:background="@color/color_def_transparent"
```

Comment:
We can only use values from `component_colors.xml` file in any layout file.

You will need to create a `color_palette.xml* resource dark/light both and then use that in `component_colors.xml` and finally use that in here. 

Have a look at this https://github.com/oppia/oppia-android/wiki/Dark-Mode

Reply:
Okay I'll do it.